zstd_stream_test.go

```diff
@@ -19,10 +20,14 @@ func failOnError(t *testing.T, msg string, err error) {
 	}
 }
 
-func testCompressionDecompression(t *testing.T, dict []byte, payload []byte) {
+func testCompressionDecompression(t *testing.T, dict []byte, payload []byte, nbWorkers int) {
 	var w bytes.Buffer
 	writer := NewWriterLevelDict(&w, DefaultCompression, dict)
-	_, err := writer.Write(payload)
+
+	err := writer.SetNbWorkers(nbWorkers)
+	failOnError(t, "Failed writing to compress object", err)
```

Comment:
as mentioned, you might want to skip the test (for nbWorkers > 1) if the error is that the library doesn't support multi-threading

Reply:
I think this is addressed.